homeassistant/components/zcs_azzurro_inverter/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "client": "Client",
+          "thing_serial": "Thing serial",
+          "name": "Friendly name"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
You don't seem to check do in config_flow. You probably should

Reply:
Sorry, I did not get what I should check in config_flow. These are there because of scaffold as well, I did not change anything